site/en/docs/web-platform/compute-pressure/index.md

```diff
@@ -1,264 +1,221 @@
-membership, the value reported to the callback is the average between
-the lower and upper bounds for the bucket.
-
-<figure>
-  {% Img src="image/FNkVSAX8UDTTQWQkKftSgGe9clO2/VUrr3MykViLDtHuTkjYG.svg", alt="A diagram illustrating the bucketing for different actual CPU utilizations", width="800", height="549" %}
-  <figcaption>
-    This diagram illustrates the relationship between three thresholds and the
-    four buckets that they form.
-  </figcaption>
-</figure>
-
-Here's how some "real" CPU utilization values, as reported by the operating
-system to the browser, would be represented as values passed to your callback:
-
-- "Real" 0% utilization would map to the 0.0 to 0.5 bucket, and would be
-  reported as `cpuUtilization: 0.25`, since 0.25 is the average of 0.0 and 0.5.
-- "Real" 40% utilization would similarly map to the 0.0 to 0.5 bucket, and be
-  reported as `cpuUtilization: 0.25`
-- "Real" 60% utilization would map to the 0.5 to 0.75 bucket, and be reported as
-  `cpuUtilization: 0.625`
-- "Real" 80% utilization would map to the 0.75 to 0.9 bucket, and be reported as
-  `cpuUtilization: 0.825`
-
-The same threshold and bucket system applies to the exposed CPU speed values.
-
-{% Aside %}
-The values reported by the API reflect the overall state of the CPU,
-due to utilization across all applications currently running. Therefore, even if
-your own web application is almost entirely idle, you may see high utilization
-and clock speed values reported, due to the activities of other applications.
-{% endAside %}
-
-## Try a demo
-
-When accessed on a browser that supports the origin trial (currently Chrome 92
-or later), [this demo](https://googlechrome.github.io/samples/compute-pressure/index.html)
-will display an ongoing readout of your device's CPU utilization and speed.
-
-
-## Acknowledgements
-
-This blog post is derived on the sample code and
-[API explainer](https://github.com/oyiptong/compute-pressure/blob/main/README.md)
-created by Olivier Yiptong and Victor Costan.
-
-The hero image was created by Robert Anasch on
-[Unsplash](https://unsplash.com/photos/-C7IKRBZHrg). The diagram illustrating
-bucketing was created by [Adam Argyle](/authors/adamargyle/).
+In this example the `cpu` is the pressure source we are interested in. For now, it is the only
+source available. In the future, there may be other sources such as `gpu`, `power`.
+
+To stop observing a source, use the `unobserve()` method, as in the following example:
+
+```js
+observer.unobserve('cpu');
+```
+
+In order to unobserve all sources at once, use the `disconnect()` method, as in the following example:
+
+```js
+observer.disconnect();
+```
+### Retrieving pressure records
+The `takeRecords()` method of the `PressureObserver` interface returns an array of
+`PressureRecords' objects stored in the pressure observer, emptying it out.
+
+Calling this method clears the pending records list, so the callback will not be run.
```

Comment:
You're defining a callback function that will not be called. There may be a better way to show in the JS example.

Reply:
See above